pkgs/top-level/all-packages.nix

```diff
@@ -32280,6 +32280,8 @@ with pkgs;
 
   eclair = callPackage ../applications/blockchains/eclair { };
 
+  eclint = callPackage ../development/tools/eclint { };
```

Comment:
all-packages is a bit of a mess be this shouldn't be added in the blockchain section, could go above `editorconfig-checker` instead.